util/chplenv/chpl_compiler.py

```diff
@@ -23,12 +23,6 @@ def validate_compiler(compiler_val, flag):
         if not os.path.isfile(comp_makefile):
             warning('Unknown compiler: "{0}"'.format(compiler_val))
 
-    if chpl_locale_model.get() == 'gpu' and flag == 'target':
```

Comment:
Is it safe to remove this? Do we still error out if we're in this invalid state:

`CHPL_GPU_LOCALE_MODEL=gpu`, `CHPL_GPU!=cpu`, and `CHPL_TARGET_COMPILER!=llvm`?

Reply:
Yes, because this is now checked in `chpl_gpu.validate` instead. https://github.com/chapel-lang/chapel/pull/22172/files#diff-4c6fe7e6f01dadbf3eba9ac00e66c1c6546abaed9ed14bf5da46f278c7892bb4R283